client/packages/common/src/api/GqlContext.tsx

```diff
@@ -196,7 +189,8 @@ export const GqlProvider: FC<PropsWithChildren<ApiProviderProps>> = ({
   );
 
   useEffect(() => {
-    setApi(createGql(url, skipRequest));
+    if (skipRequest) client.setSkipRequest(skipRequest);
+    if (url) client.setEndpoint(url);
   }, [url, skipRequest]);
```

Comment:
Not sure if this `useEffect` is actually needed at all? The `url` shouldn't change, but I'm not sure when `skipRequest`might.